pages/toolkits/entertainment/twitch.mdx

```diff
@@ -0,0 +1,118 @@
+import { Tabs } from "nextra/components";
+
+# Twitch auth provider
+
+<Note>
+  At this time, Arcade does not offer a default Twitch Auth Provider. To use
+  Twitch auth, you must create a custom Auth Provider with your own Twitch
+  OAuth 2.0 credentials as described below.
+</Note>
+
+The Twitch auth provider enables tools and agents to call the Twitch API on behalf of a user. Behind the scenes, the Arcade Engine and the Twitch auth provider seamlessly manage Twitch OAuth 2.0 authorization for your users.
+
+### What's documented here
+
+This page describes how to use and configure Twitch auth with Arcade.
+
+This auth provider is used by:
+
+- Your [app code](#using-twitch-auth-in-app-code) that needs to call Twitch APIs
+- Or, your [custom tools](#using-twitch-auth-in-custom-tools) that need to call Twitch APIs
+
+## Configuring Twitch auth
+
+### Create a Twitch app
+
+- Twitch requires that you have two-factor authentication enabled for your account. Enable in your [account security seetings](https://www.twitch.tv/settings/security)
+- Create a Twitch Application in the [Twitch App Console](https://dev.twitch.tv/console/apps)
+- Set the OAuth Redirect URL to: `https://cloud.arcade.dev/api/v1/oauth/callback`
+- Select your Application category
+- Select the 'Confidential' Client Type
```

Comment:
Should this be 'Confidential' client type or 'Public'?

<img width="1236" alt="Screenshot 2025-03-17 at 1 03 24 PM" src="https://github.com/user-attachments/assets/15a1ac4c-d3ed-4320-a9ce-b16ca3806c8f" />

Reply:
Confidential